Lib/test/test_inspect.py

```diff
@@ -57,6 +57,25 @@ def revise(filename, *args):
 
 git = mod.StupidGit()
 
+
+def signatures_with_lexigraphic_keyword_only_parameters():
+    """
+    Yields a whole bunch of functions with only keyword-only parameters,
+    where those parameters are always in lexigraphically sorted order.
+    """
+    parameters = ['a', 'bar', 'c', 'delta', 'ephraim', 'magical', 'yoyo', 'z']
```

Comment:
I realize why you're generating parameters in sorted order here, but I think at least one test with non-sorted parameters might be a good idea, just to show the principle. Maybe also specifying them in reverse sorted order would be good enough?

Reply:
I'm not sure what you're proposing here.  You want one test to fail, just to show that the detection mechanism is working?  Or, you want me to hand-code one test where the parameters aren't sorted, just in case something in Python was sorting parameters?

Reply:
No, I want one test to pass that doesn't have the parameters in lexical order.

Reply:
Done.